src/modality-specific-files/magnetic-resonance-imaging-data.md

```diff
@@ -207,6 +219,18 @@ entity corresponds to modality suffix,
 such as `T1w` or `inplaneT1`, referenced by the defacemask image.
 For example, `sub-01_mod-T1w_defacemask.nii.gz`.
 
+<!--
+This block generates a filename templates.
+The inputs for this macro can be found in the directory
+  src/schema/rules/files/raw
+and a guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_filename_template("raw", datatypes=["anat"], suffixes=[
+         "defacemask",
+      ])
+}}
+
 The OPTIONAL [`task-<label>`](../appendices/entities.md#task) entity can be used
 in order to allow tasks during structural MR acquisitions,
 for example pre-described motion paradigms such as nodding, to be described.
```

Comment:
Maybe this could go after the parametric suffixes section.

Reply:
done